nodejs/awsx/lb/loadBalancer.ts

```diff
@@ -39,7 +39,7 @@ export abstract class LoadBalancer extends pulumi.ComponentResource {
         // people didn't have direct control over creating the LB.  In awsx though creating the LB
         // is easy to do, so we just let the user pass in the name they want.  We simply add an
         // alias from the old name to the new one to keep things from being recreated.
-        this.loadBalancer = new aws.lb.LoadBalancer(name, {
+        this.loadBalancer = args.loadBalancer || new aws.lb.LoadBalancer(name, {
             ...args,
             subnets: getSubnets(args, this.vpc, external),
             internal: external.apply(ex => !ex),
```

Comment:
i have a few concerns here.  Namely that this arg overrides other args passed in.  For example, if the user provides this LB, then i think we need to do things like check that `args.external` is *not passed in*.  Otherwise they can be in conflict.  For example, what would it mean to say "make me an internal awsx.LB out of this external aws.LB"?

Reply:
Makes sense. Will try take a look soon